src/Microsoft.SourceIndexer.Tasks/DownloadStage1Index.cs

```diff
@@ -14,7 +15,13 @@ namespace Microsoft.SourceIndexer.Tasks
 {
     public class DownloadStage1Index : Task
     {
-        public string BlobContainerSasUrl { get; set; }
+        public string ClientId { get; set; }
+
+        [Required]
+        public string StorageAccount { get; set; }
```

Comment:
Maybe add a comment about the supported values like `https://netsourceindexstage1.blob.core.windows.net/` and `netsourceindexstage1`.